src/FsCodec.Box/TryDeflate.fs

```diff
@@ -9,9 +9,17 @@ module private EncodedMaybeDeflated =
     type Encoding =
         | Direct = 0
         | Deflate = 1
+        | Brotli = 2
     type Encoded = (struct (int * ReadOnlyMemory<byte>))
     let empty : Encoded = int Encoding.Direct, ReadOnlyMemory.Empty
 
+    let private brotliDecompress (data: ReadOnlyMemory<byte>): byte[] =
+        use s = new System.IO.MemoryStream(data.ToArray())
```

Comment:
on L26 I have writable=false - I think I was being explicit for some reason?

Reply:
I'm still a bit confused/worried by the variations (will something blow up in some obscure DDB scenario due to the deviation in `leaveOpen`+`let` vs `use` and not specifying `leaveOpen`, but I'm going to let it go)
It's definitely clear from reading [the brotli impl](https://github.com/dotnet/runtime/tree/main/src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression) that the `use` / `Dispose` of the `compressor` and `decompressor` is important for returning stuff to buffer pools etc.